ci/test_mac.sh

```diff
@@ -4,7 +4,6 @@
 #
 set -o pipefail
 THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-source $THIS_DIR/_init.sh
```

Comment:
i don't think we should change mac build here

Reply:
ok